src/RequestClientWrapper.ts

```diff
@@ -117,16 +112,9 @@ export default class RequestClientWrapper {
     if (options.headers === undefined) {
       options.headers = {};
     }
-    this.setSupabaseEnv(options.baseURL);
     if (this.isJwtToken) {
-      let accessToken = this.requestClientWrapperConfiguration.session;
-      const session = await this.checkSupabaseSession();
-      if (session) {
-        accessToken = session.access_token;
-      } else if (this.isJwtTokenExpired(accessToken)) {
```

Comment:
I think it could be beneficial to possibly throw a runtime exception here to prevent the API call from reaching our API and give the end user a better message up front on why this request will fail. I'm not strongly opinionated on that as the API will throw a similar error and we know React won't give us one that is expired. Just trying to think of saving the network on our API and lambdas as well though.